### DISCUSSION TITLE
Install on windows without package manager?

### DISCUSSION THREAD

**Discussion by pancake-tongs** [VSCodium/vscodium#1410] [Q&A] [answered]
I'm very interested in switching from code to codium, but I don't use any of the 3 supported package managers. I've used several of them before but stopped in each case because in my experience they tend to introduce more problems than they solve for most win32 apps. Is there a documented way to install Codium without requiring the use of a third party package management system?

**Comment by daiyam** [comment, marked as answer]
Install with a binary found at https://github.com/VSCodium/vscodium/releases/tag/1.75.1.23040

**Reply by apitofme** [reply]
[This discussion](https://github.com/VSCodium/vscodium/discussions/1214) may help to add some clarity as to which 'asset' file to download for Windows users.

TLDR / Notes:
- _'VSCodiumUserSetup-*.exe'_ is the recommended install method (i.e. single-user install - no admin; allows for "smoother" updates)
- My assumption from the linked discussion is that the _.msi_ installer will allow you to choose between User/System level at install time
- [Portable installations](https://code.visualstudio.com/docs/editor/portable) are **only** possible with the _zip file_ versions and require additional steps after extracting the archive (namely creating a "data" folder in the extracted folder structure)

**Warning: I have not tested any of the aforementioned files or assumptions!!**

#1214
